src/view/screens/Search/Search.tsx

```diff
@@ -316,8 +324,33 @@ function SearchScreenUserResults({query}: {query: string}) {
   )
 }
 
-const SECTIONS_LOGGEDOUT = ['Users']
-const SECTIONS_LOGGEDIN = ['Posts', 'Users']
+const SECTIONS_LOGGEDOUT = ['PEOPLE'] as const
```

Comment:
I think we’ll want to open up posts search logged out too. Let’s keep track of this somewhere.